README.md

```diff
@@ -121,4 +121,19 @@ vendor/bin/roave-backward-compatibility-check --help
 
 ## Configuration
 
-There are currently no configuration options available.
+The file `.roave-backward-compatibility-check.json` is read from the current working directory (when it exists) and sets configuration for the command.
```

Comment:
We can also change the configuration file name. My goal was to not think too much about it while leaving room for adding new properties.